apollo-audit/apollo-audit-spring-boot-starter/src/main/java/com/ctrip/framework/apollo/audit/aop/ApolloAuditSpanAspect.java

```diff
@@ -87,14 +86,8 @@ public Object around(ProceedingJoinPoint pjp, ApolloAuditLog auditLog) throws Th
 
     try (AutoCloseable scope = api.appendSpan(auditLog.type(), auditLog.name(),
         auditLog.description())) {
+      dataInfluenceList.forEach(e -> api.appendDataInfluenceWrapper(e.getClass()));
       returnVal = pjp.proceed();
```

Comment:
Can we append DataInfluence after method finished?

Maybe after `pjp.proceed()` finished, just call a method in api will be more simpler?

for example

```java
returnVal = pjp.proceed();
Class<?> xxxEntity = // get from method signaure, argument with annotation
api.appendDataInfluenceList(xxxEntity, dataInfluenceList);
```

and it work will without appendDataInfluenceWrapper?